manager/controllers/default/resource/update.class.php

```diff
@@ -119,7 +119,7 @@ public function process(array $scriptProperties = array()) {
         $this->resourceArray['cacheable'] = intval($this->resourceArray['cacheable']) == 1 ? true : false;
         $this->resourceArray['deleted'] = intval($this->resourceArray['deleted']) == 1 ? true : false;
         $this->resourceArray['uri_override'] = intval($this->resourceArray['uri_override']) == 1 ? true : false;
-        $this->resourceArray['syncsite'] = isset($this->resourceArray['syncsite']) && intval($this->resourceArray['syncsite']) == 1 ? true : $this->context->getOption('syncsite_default', 1, $this->modx->_userConfig);
+        $this->resourceArray['syncsite'] = intval($this->context->getOption('syncsite_default', 1, $this->modx->_userConfig)) == 1 ? true : false;
```

Comment:
My understanding is that this won't take care of form customization.
You might want to check my proposed [PR](https://github.com/modxcms/revolution/pull/12218) (which "wrong behavior" was trying to get the resource context while it was already available, see https://github.com/modxcms/revolution/issues/12275#issuecomment-71428150)